parts/common/components.json

```diff
@@ -1194,23 +1194,56 @@
       }
     },
     {
-      "name": "cri-tools",
+      "name": "kubernetes-cri-tools",
       "downloadLocation": "/opt/crictl/downloads",
       "downloadURIs": {
         "windows": {
           "default": {
             "versionsV2": []
           }
         },
+        "ubuntu": {
+          "r1804": {
+            "versionsV2": [
+              {
+                "renovateTag": "name=kubernetes-cri-tools, os=ubuntu, release=18.04",
+                "latestVersion": "1.32.0-ubuntu18.04u3"
+              }
+            ]
+          },
+          "r2004": {
+            "versionsV2": [
+              {
+                "renovateTag": "name=kubernetes-cri-tools, os=ubuntu, release=20.04",
+                "latestVersion": "1.32.0-ubuntu20.04u3"
+              }
+            ]
+          },
+          "r2204": {
+            "versionsV2": [
+              {
+                "renovateTag": "name=kubernetes-cri-tools, os=ubuntu, release=22.04",
+                "latestVersion": "1.32.0-ubuntu22.04u3"
+              }
+            ]
+          },
+          "r2404": {
+            "versionsV2": [
+              {
+                "renovateTag": "name=kubernetes-cri-tools, os=ubuntu, release=24.04",
+                "latestVersion": "1.32.0-ubuntu24.04u3"
+              }
+            ]
+          }
+        },
         "default": {
           "current": {
             "versionsV2": [
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
```

Comment:
we dont support AzLinux yet ?

Reply:
For Renovate, we still need to add a customer manager to detect the mariner repo. I can work on it after this PR.
FWIW, now in components.json Default.current means everything else, which includes Mariner v2 and AzureLinux v2 and v3.